link_checker.py

```diff
@@ -93,37 +93,37 @@ def parse_argument(args):
         LOCAL = True
 
 
-def get_local_license():
+def get_local_licenses():
     """This function get all the licenses stored locally
 
     Returns:
         list: list of file names of license file
     """
     try:
-        all_files = os.listdir(LICENSE_LOCAL_PATH)
+        license_names_unordered = os.listdir(LICENSE_LOCAL_PATH)
     except FileNotFoundError:
         raise CheckerError(
             "Local license path({}) does not exist".format(LICENSE_LOCAL_PATH)
         )
     # Catching permission denied(OS ERROR) or other errors
     except:
         raise
-    links_ordered = list()
+    license_names_unordered.sort()
```

Comment:
it's a little confusing that a variable named `unordered` is being sorted in place, it becomes no longer unordered. could you make it clearer?